docs/reference/commandline/pause.md

```diff
@@ -24,13 +24,17 @@ Options:
       --help   Print usage
 ```
 
-The `docker pause` command suspends all processes in a container. On Linux,
-this uses the cgroups freezer. Traditionally, when suspending a process the
-`SIGSTOP` signal is used, which is observable by the process being suspended.
+The `docker pause` command suspends all processes the specified container.
```

Comment:
@thaJeztah PTAL
I think that it is good to use "in the specified containers." better than "in a container".
Updated this PR.